From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Nicolas Iooss <nicolas.iooss_linux@m4x.org>
Date: Sat, 27 Feb 2021 16:28:02 +0100
Subject: [PATCH] {MAYBE UPS} drivers/platform/surface/aggregator: ACPI_HANDLE
 returns a value

As acpi_handle is a void*, this does not change anything, but this
helps the deref checker plugin.
---
 drivers/platform/surface/aggregator/core.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/platform/surface/aggregator/core.c b/drivers/platform/surface/aggregator/core.c
index 8dc2c267bcd6..2156309d638a 100644
--- a/drivers/platform/surface/aggregator/core.c
+++ b/drivers/platform/surface/aggregator/core.c
@@ -622,7 +622,7 @@ static const struct acpi_gpio_mapping ssam_acpi_gpios[] = {
 static int ssam_serial_hub_probe(struct serdev_device *serdev)
 {
 	struct ssam_controller *ctrl;
-	acpi_handle *ssh = ACPI_HANDLE(&serdev->dev);
+	acpi_handle ssh = ACPI_HANDLE(&serdev->dev);
 	acpi_status astatus;
 	int status;
 
-- 
